main.py

```diff
@@ -22,105 +16,13 @@
 from flaskext.markdown import Markdown
 from icalendar import Calendar, Event
 
+from miniconf.load_site_data import load_site_data
+from miniconf.site_data import Poster, Tutorial, Workshop
+from miniconf.utils import format_paper
+
 site_data = {}
 by_uid = {}
 qa_session_length_hr = 1
-merged_committees: List[object] = []  # type "List[object]"
```

Comment:
we will just correct the `committee.yml` file without introducing this python logic